colors/material.vim

```diff
@@ -386,6 +386,9 @@ hi link SignifySignDelete GitGutterDelete
 " vim-better-whitespace
 call s:SetHighlight('ExtraWhitespace', '', s:red, '')
 
+" NERDTree
+call s:SetHighlight('NERDTreeFile', s:fg, '', '')
```

Comment:
It looks like this is only an issue in Neovim (I suspect it's either a bug in NERDTree or Neovim). Can we scope this change to Neovim only?

```vim
" NERDTree
if has('nvim')
  call s:SetHighlight('NERDTreeFile', s:fg, '', '')
endif
```